src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -2170,6 +2170,10 @@ public partial class DbEnumerator : System.Collections.IEnumerator
     }
     public abstract partial class DbException : System.Runtime.InteropServices.ExternalException
     {
+        /// <summary>
+        /// When overriden in a derived class return true in all cases where a simple retry of the operation without any other change may be successful.
+        /// </summary>
+        public virtual bool IsTransient { get; protected set; }
```

Comment:
Please define this property like other properties in the ref file:

```c#
public virtual bool IsTransient { get { throw null; } }
```

This isn't an implementation, just a contract - so it only needs to compile. Also, please move this down under the constructors as is the convention with other types here.